iOS/RemoteCollectionController.swift

```diff
@@ -2,9 +2,25 @@ import UIKit
 
 class RemoteCollectionController: UICollectionViewController {
     var sections = Photo.constructRemoteElements()
-    let sectionScrubber = SectionScrubber()
-    var keyWindow: UIWindow?
-    let overlayView = UIView(frame: UIScreen.mainScreen().bounds)
+
+    lazy var overlayView: UIView = {
+        let view = UIView(frame: UIScreen.mainScreen().bounds)
+        view.backgroundColor = UIColor.blackColor()
+        view.alpha = 0.0
+
+        return view
+    }()
+
+    lazy var sectionScrubber: SectionScrubber = {
+        let scrubber = SectionScrubber(collectionView: self.collectionView!)
+        scrubber.delegate = self
+        scrubber.scrubberImage = UIImage(named: "date-scrubber")
+        scrubber.sectionLabelImage = UIImage(named: "section-label")
+        scrubber.sectionLabelFont = UIFont(name: "DINNextLTPro-Light", size: 18)
+        scrubber.sectionlabelTextColor = UIColor(red: 69/255, green: 67/255, blue: 76/255, alpha: 0.8)
```

Comment:
As a standalone component, I feel this should be configured outside, from the installing application.